src/main/java/org/sagebionetworks/bridge/udd/config/SpringConfig.java

```diff
@@ -74,52 +100,31 @@ public DynamoQueryHelper ddbQueryHelper() {
 
     @Bean(name = "ddbStudyTable")
     public Table ddbStudyTable() {
-        return ddbClient().getTable(ddbPrefix() + "Study");
+        return ddbClient().getTable(ddbPrefix + "Study");
     }
 
     @Bean(name = "ddbSynapseMapTable")
     public Table ddbSynapseMapTable() {
-        return ddbClient().getTable(environmentConfig().get("synapse.map.table"));
+        return ddbClient().getTable(envConfig.getProperty(envConfig.getProperty("bridge.env") + ".synapse.map.table"));
```

Comment:
We have a lot of config properties that get overridden by env-specific properties. Instead of needing to write this code every time we consume a config property, can we bake it into the config system?